.github/workflows/release.yaml

```diff
@@ -0,0 +1,34 @@
+name: Release Package
+
+on: push
+
+jobs:
+  pypi-release:
+    # run only on tags
+    if: startsWith(github.ref, 'refs/tags')
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v1
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: python -m pip install --upgrade pip wheel
+
+      - name: Build Dist
+        run: python setup.py sdist bdist_wheel
+
+      - name: Publish package
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: __token__
+          password: ${{ secrets.pypi_password }}
```

Comment:
`${{ secrets.PYPI_API_TOKEN }}`

Reply:
Can we use test.pypi for now (with `PYPI_TEST_API_TOKEN`) and once verified OK there change to regular pypi?